### ISSUE TITLE
Restart is broken in the master branch

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-simulators#1843]
This should be a regression. I was using this functionality some weeks ago (or couple of months).

```
flow: /home/kaib/OPM-devel2/debug/opm-simulators/ebos/ecloutputblackoilmodule.hh:1017: void Ewoms::EclOutputBlackOilModule<TypeTag>::setRestart(const Opm::data::Solution&, unsigned int, unsigned int) [with TypeTag = Ewoms::Properties::TTag::EclFlowProblem]: Assertion `saturation_[oilPhaseIdx].size() > 0' failed.
```

It looks like the vector of `saturation_` is not properly sized. Not sure since when this has been broken.

**Comment by bska** [comment]
Possibly related to #1828.

**Comment by GitPaean** [comment]
I did not delete `EQUIL` related, while in #1828, it runs through with `EQUIL` related.

**Comment by GitPaean** [comment]
It is possible the `UNRST` does not contain enough data for RESTART .  I saw `NORST=1` in the `RPTRST` setup. I am rerunning to confirm this. 

Sorry for potential noise.

**Comment by bska** [comment]
> I saw `NORST=1` in the `RPTRST` setup.

Flow [does not support](https://github.com/OPM/opm-common/blob/d55433431625eb484a54fa19de8f7be2f9b5c2b7/src/opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp#L73) `NORST`.

The only reason I mentioned #1828 is that the two issues **might** have the same underlying cause.

**Comment by GitPaean** [comment]
> Flow does not support NORST.

Thanks. I thought of it. Anyway, I am re-running to confirm the problem.

**Comment by GitPaean** [comment]
Re-running encountered the same problem.

**Comment by totto82** [comment]
Already fixed in #1839

**Comment by GitPaean** [comment]
> Already fixed in #1839

Good to know. Thanks.

**Comment by GitPaean** [comment]
Closed by #1839.
